src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx

```diff
@@ -208,4 +208,36 @@ Press Ignore to ignore the external changes. Your changes may be lost if you clo
   <data name="AnalyzersNodeName" xml:space="preserve">
```

Comment:
Nevermind, hadn't gotten to the part where you added Resources to the Managed project yet.